src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -220,6 +220,293 @@ int16_t EnforceCoolingSetpointLimits(int16_t CoolingSetpoint, EndpointId endpoin
     return CoolingSetpoint;
 }
 
+struct SetpointLimits
+{
+    bool AutoSupported      = false;
+    bool HeatSupported      = false;
+    bool CoolSupported      = false;
+    bool OccupancySupported = false;
+
+    int16_t AbsMinHeatSetpointLimit;
+    int16_t AbsMaxHeatSetpointLimit;
+    int16_t MinHeatSetpointLimit;
+    int16_t MaxHeatSetpointLimit;
+    int16_t AbsMinCoolSetpointLimit;
+    int16_t AbsMaxCoolSetpointLimit;
+    int16_t MinCoolSetpointLimit;
+    int16_t MaxCoolSetpointLimit;
+    int8_t DeadBand      = 0;
+    int16_t DeadBandTemp = 0;
+    int16_t OccupiedCoolingSetpoint;
+    int16_t OccupiedHeatingSetpoint;
+    int16_t UnoccupiedCoolingSetpoint;
+    int16_t UnoccupiedHeatingSetpoint;
```

Comment:
usual convention for struct members is argument names/variable names (e.g. `absMinHeatSetpointLimit`)

Reply:
Fixed.